edm4hep.yaml

```diff
@@ -489,3 +505,72 @@ datatypes :
     OneToOneRelations:
      - edm4hep::ReconstructedParticle rec // reference to the reconstructed particle
      - edm4hep::Vertex vertex             // reference to the vertex
+
+
+  #----------- SimPrimaryIonizationCluster
+  edm4hep::SimPrimaryIonizationCluster:
+    Description: "Simulated Primary Ionization"
+    Author : "Wenxing Fang, IHEP"
+    Members:
+      - uint64_t cellID                      //cell id.
+      - float time                           //the primary ionization's time in the lab frame in [ns].
+      - edm4hep::Vector3d position           //the primary ionization's position in [mm].
+      - int16_t type                         //type.
+    VectorMembers:
+       - uint64_t electronCellID             //cell id.
+       - float electronTime                  //the time in the lab frame in [ns].
+       - edm4hep::Vector3d electronPosition  //the position in the lab frame [mm].
+       - float pulseTime                     //the pulse's time in the lab frame in [ns].
+       - float pulseAmplitude                //the pulse's amplitude.
+    OneToOneRelations:
+      - edm4hep::MCParticle MCParticle       //the particle that caused the ionizing collisions.
+
+ #----------  TrackerPulse
+  edm4hep::TrackerPulse:
+    Description: "Reconstructed Tracker Pulse"
+    Author : "Wenxing Fang, IHEP"
+    Members:
+       - uint64_t cellID                 //cell id.
+       - float time                      //time.
+       - float charge                    //charge.
```

Comment:
The charge is in coulomb or multiple or electron (or proton) charges?

Reply:
It is in coulomb.